rand_core/src/lib.rs

```diff
@@ -299,6 +299,12 @@ pub trait SeedableRng: Sized {
     /// reference implementation of the PRNG given equivalent seed; if necessary
     /// another constructor replicating behaviour from a reference
     /// implementation can be added.
+    ///
+    /// `from_seed` should never panic. Some PRNGs cannot use certain seeds, for
+    /// example variants of the Xorshift family don't work with an all zero
+    /// seed. In such cases it is recommended to replace the bad seed with a
```

Comment:
I don't see the need to mention Xorshift explicitly here; maybe say something like:

> In the case that some special values (e.g. 0) are not viable seeds it is preferable to map these to alternative constant value(s) (e.g. `0xBAD5EEDu32`); this is assuming only a small number of values must be rejected.